addons/website/data/website_demo.xml

```diff
@@ -409,5 +409,67 @@
             <field name="is_published">False</field>
             <field name="view_id" ref="snippets_debug_view"/>
         </record>
+
+    </data>
+    <data>
```

Comment:
why inside a new data node ?  First one is already a data node without the 'no update'